ui-admin/src/portal/siteContent/SiteContentEditor.tsx

```diff
@@ -53,20 +55,38 @@ const SiteContentEditor = (props: InitializedSiteContentViewProps) => {
     setWorkingContent(newWorkingContent)
   }
 
+  const insertNewPage = (page: HtmlPage) => {
+    if (!localContent) {
+      return
+    }
+    const newNavBarItem: NavbarItemInternal = {
```

Comment:
Just curious, what's the difference between NavBarItems of type `EXTERNAL`, `INTERNAL`, etc.?

Reply:
Check out NavbarItemType.java for the enum with comments